test/minter-suite-minters/SetPrice/MinterSetPrice/MinterSetPriceV2.test.ts

```diff
@@ -19,8 +19,8 @@ import {
 } from "../../../util/common";
 
 import { MinterSetPrice_ETH_Common } from "./MinterSetPrice.common";
-import { MinterSetPriceV1V2_Common } from "../MinterSetPriceV1V2.common";
-import { MinterSetPriceV2_Common } from "../MinterSetPriceV2.common";
+import { MinterSetPriceV1V2V3_Common } from "../MinterSetPriceV1V2V3.common";
+import { MinterSetPriceV2V3_Common } from "../MinterSetPriceV2V3.common";
```

Comment:
I think this import is missing, which, tbh, I'm really surprised isn't causing this test to fail.
IIUC, the fix here would be to update the export name from `MinterSetPriceV2_Common` to `MinterSetPriceV2V3_Common` in `"../MinterSetPriceV2V3.common"`

Reply:
ah geez, literally had this in my editor updated but just forgot to hit save on the file before pushing 😅 but ya super weird it didn't actually cause a test to fail on CI